README.md

```diff
@@ -47,45 +51,59 @@ Substrate's canonical mnemonic: `bottom drive obey lake curtain smoke basket hol
 
 # Charleth:
-- Address:0x798d4Ba9baf0064Ec19eB4F0a1a45785ae9D6DFc
-- PrivKey:0x0b6e18cafb6ed99687ec547bd28139cafdd2bffe70e6b688025de6b445aa5c5b
+- Address: 0x798d4Ba9baf0064Ec19eB4F0a1a45785ae9D6DFc
+- PrivKey: 0x0b6e18cafb6ed99687ec547bd28139cafdd2bffe70e6b688025de6b445aa5c5b
 
 # Dorothy:
-- Address:0x773539d4Ac0e786233D90A233654ccEE26a613D9
-- PrivKey:0x39539ab1876910bbf3a223d84a29e28f1cb4e2e456503e7e91ed39b2e7223d68
+- Address: 0x773539d4Ac0e786233D90A233654ccEE26a613D9
+- PrivKey: 0x39539ab1876910bbf3a223d84a29e28f1cb4e2e456503e7e91ed39b2e7223d68
 
 # Ethan:
-- Address:0xFf64d3F6efE2317EE2807d223a0Bdc4c0c49dfDB
-- PrivKey:0x7dce9bc8babb68fec1409be38c8e1a52650206a7ed90ff956ae8a6d15eeaaef4
+- Address: 0xFf64d3F6efE2317EE2807d223a0Bdc4c0c49dfDB
+- PrivKey: 0x7dce9bc8babb68fec1409be38c8e1a52650206a7ed90ff956ae8a6d15eeaaef4
 
 # Faith:
-- Address:0xC0F0f4ab324C46e55D02D0033343B4Be8A55532d
-- PrivKey:0xb9d2ea9a615f3165812e8d44de0d24da9bbd164b65c4f0573e1ce2c8dbd9c8df
+- Address: 0xC0F0f4ab324C46e55D02D0033343B4Be8A55532d
+- PrivKey: 0xb9d2ea9a615f3165812e8d44de0d24da9bbd164b65c4f0573e1ce2c8dbd9c8df
 
 # Gerald:
-- Address:0x7BF369283338E12C90514468aa3868A551AB2929
-- PrivKey:0x96b8a38e12e1a31dee1eab2fffdf9d9990045f5b37e44d8cc27766ef294acf18
+- Address: 0x7BF369283338E12C90514468aa3868A551AB2929
+- PrivKey: 0x96b8a38e12e1a31dee1eab2fffdf9d9990045f5b37e44d8cc27766ef294acf18
+
+# Heath:
+- Address: 0x931f3600a299fd9B24cEfB3BfF79388D19804BeA
+- PrivKey: 0x0d6dcaaef49272a5411896be8ad16c01c35d6f8c18873387b71fbc734759b0ab
+
+# Ida:
+- Address: 0xC41C5F1123ECCd5ce233578B2e7ebd5693869d73
+- PrivKey: 0x4c42532034540267bf568198ccec4cb822a025da542861fcb146a5fab6433ff8
+
+# Judith:
+- Address: 0x2898FE7a42Be376C8BC7AF536A940F7Fd5aDd423
+- PrivKey: 0x94c49300a58d576011096bcb006aa06f5a91b34b4383891e8029c21dc39fbb8b
 ```
 
 ## Build the Moonbeam Node
 
-To build Moonbeam, you will need a proper Substrate development environment. If you've never worked
-with a Substrate-based blockchain before, you should probably try the [Setting Up a Moonbeam Node](https://docs.moonbeam.network/getting-started/local-node/setting-up-a-node/) docs first. If you
-need a refresher setting up your Substrate environment, see [Substrate's Getting Started Guide](https://substrate.dev/docs/en/knowledgebase/getting-started/).
+To build Moonbeam, you will need a proper Substrate development environment. If you've never worked with a Substrate-based blockchain before, you should probably try the [Setting Up a Moonbeam Node](https://docs.moonbeam.network/getting-started/local-node/setting-up-a-node/) docs first. 
+If you need a refresher setting up your Substrate environment, see [Substrate's Getting Started Guide](https://substrate.dev/docs/en/knowledgebase/getting-started/).
 
 ```bash
 # Fetch the code
 git clone https://github.com/PureStake/moonbeam
 cd moonbeam
 
+# Checkout the latest release tag
+git checkout tags/$(git tag | tail -1)
+
```

Comment:
I'm personally not a fan of this. I think it makes sense in docs or tutorials that are targeting a particular release. But to me this readme should explain how to build the exact code from _this commit you're reading_

Reply:
Yeah I understand. I'll change this but I'll add a note to say that `master` might be unstable or something like that.

I wanted to add this because we've had people just cloning master and then their stuff is not working

Reply:
> we've had people just cloning master and then their stuff is not working

It's been a goal of mine that master should always be able to sync the public chains. So far that hasn't been priority, and we've worked around it by purging the public chains all the time. But for long-lived networks like moonriver, we'll need to get this straight. So hopefully it will be less of an issue then.

Reply:
`master` has broken from time to time, but I share your goal of keeping it in good shape.

`git log | tail -1` isn't guaranteed to give any useful result -- I could push a tag to an arbitrary commit right now and this would do weird things.

Maybe some instructions on how to identify the most recent tag (something like "look at the tag mentioned in Moonbeam's most recent release")

Reply:
Yeah understood, although the idea of tags is to create them for stable releases and so on haha. But I just added a note that building `master` might provide an unstable build and so on, with a link to the latest releases.